dropwizard-logging/src/main/java/io/dropwizard/logging/LoggingUtil.java

```diff
@@ -0,0 +1,71 @@
+package io.dropwizard.logging;
+
+import ch.qos.logback.classic.LoggerContext;
+import io.dropwizard.util.Duration;
+import org.slf4j.ILoggerFactory;
+import org.slf4j.LoggerFactory;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import javax.annotation.concurrent.GuardedBy;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class LoggingUtil {
```

Comment:
I wonder should we make this class a package protected or public... I mean, do we want its methods be visible for user logger factories or should they be closed as a part of internal Dropwizard logging mechanism? I'm more apt to to the latter. What is your opinion on this?